crates/re_viewer/src/ui/selection_panel.rs

```diff
@@ -720,6 +720,35 @@ fn container_top_level_properties(
                     },
                 );
             }
+            ui.end_row();
+
+            // ---
+
+            fn equal_shares(shares: &[f32]) -> bool {
+                shares.iter().all(|&share| share == shares[0])
+            }
+
+            let all_shares_are_equal =
+                equal_shares(&container.col_shares) && equal_shares(&container.row_shares);
+
+            if container.contents.len() > 1
+                && match container.container_kind {
+                    egui_tiles::ContainerKind::Tabs => false,
+                    egui_tiles::ContainerKind::Horizontal
+                    | egui_tiles::ContainerKind::Vertical
+                    | egui_tiles::ContainerKind::Grid => true,
+                }
+                && ui
+                    .add_enabled(
+                        !all_shares_are_equal,
+                        egui::Button::new("Equalize children"),
```

Comment:
I like this feature but I don't like equalize, makes me think they're diving 😄 
What do you think of `Distribute equally`?

Kinda want to get in `shares` here because that's what it is called in the blueprint code 🤔

Reply:
I tend to start with seeing what Figma does. 

![Screenshot 2024-05-03 at 09 12 43](https://github.com/rerun-io/rerun/assets/2624717/c4852281-781c-4c6f-a8c7-f32923db0159)

They call it "Distribute vertical/horizontal spacing". I'd go for "Distribute content equally". 
```suggestion
                        egui::Button::new("Distribute content equally"),
```

Either way I don't think it matter all that much because the "right way" to do this is to have this button right next to the "shares" blueprint property setting of the container anyway. That way the context will make it much clearer anyway